src/orion/core/io/cmdline_parser.py

```diff
@@ -20,28 +20,39 @@
 
 
 class CmdlineParser(object):
+    """CmdlineParser is aimed at providing a simple tool to interpret commandline arguments
```

Comment:
There should be a short summary followed by the description. 
Numpy docstring-guide:
https://docs.scipy.org/doc/numpy/docs/howto_document.html#numpydoc-docstring-guide

Most specifically:
https://docs.scipy.org/doc/numpy/docs/howto_document.html#sections